plinth/modules/shaarli/views.py

```diff
@@ -28,6 +28,9 @@
 from plinth import package
 from plinth.modules import shaarli
 
+def on_install():
+"""Notify that the service is now enabled."""
+shaarli.service.notify_enabled(None, True)
 
 @package.required(['shaarli'])
```

Comment:
From my account, it is still line 32
http://i.imgur.com/bAI44xd.png

+def on_install(): reference is not displayed
i am not sure then if i need to copy again +def on_install():